### PR TITLE
Update nov2-team-minutes.md

### LINKED ISSUE #197: Team Meeting Minutes

### LINKED ISSUE #217: Review W9 documentations
- [x] team minutes
- [x] Client minutes
- [x] Client agenda
- [x] Unit test instructions

### PR DESCRIPTION
**Pull request by aleksssk**
updated for this weeks meeting

### CODE REVIEW

File: meeting-minutes/team-minutes/nov2-team-minutes.md [hunk 1]
```diff
@@ -5,17 +5,57 @@
...
 
 ## Detailed Notes:
+- Every meeting we are going to do a status update meeting
+- Status meeting is where everyone tells what they have been working on during the sprint and if they had any problems
+- Delete the time availability for the front end form
+- Database is setup
+- Basic front end is setup
+- Code will automatically run ESLint
+- ESLint will keep all syntax the same 
+- Run ESLint before you do a start the program so that your syntax is the same as the standard we are using
```
Comment (katmartel) [inline meeting-minutes/team-minutes/nov2-team-minutes.md]: Sorry, just to clarify: by this, do you mean run EsLint locally before we commit/push to Github?

**Review by katmartel — APPROVED** [review]
It looks like you've covered everything that we discussed during today's meeting, so great work!

File: meeting-minutes/team-minutes/nov2-team-minutes.md (hunk 1 above)
Comment (aleksssk) [reply meeting-minutes/team-minutes/nov2-team-minutes.md]: Yes, I will fix it right now to make it more clear

**Review by wrc9817 — APPROVED** [review]
good job

**Review by jpabadir — APPROVED** [review]
Great notes!

**Review by jpabadir — APPROVED** [review]
Nice detailed notes!

**Pull request merged**